src/components/Modal/Modal.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import styled from 'styled-components'
+import EscapeOutside from '../EscapeOutside/EscapeOutside'
+import { Button, Text, breakpoint } from '@aragon/ui'
+
+const WrapModal = styled(EscapeOutside)`
+  height: calc(100vh - 2em);
```

Comment:
We should definitely give you a `small` breakpoint to avoid this :smile:

<img src="https://user-images.githubusercontent.com/4166642/46490404-34724e00-c808-11e8-8162-b3229a5bf4c0.png" height=400>

Reply:
Screens smaller than `768px`? I don't think it is needed, most probably those sizes belong to smartphones.

Reply:
The reason why there is no `small` breakpoint is to ensure the base CSS is working with any size of screen (mobile first) with exceptions (breakpoints) for the larger sizes, rather than the other way. https://github.com/aragon/aragon-ui/blob/master/src/utils/styles/breakpoint.js#L7